<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjDeclSel</class>
 <widget class="QDialog" name="AdjDeclSel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Adjective Declension Selector</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>780</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Adjective Declension List</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBttn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="declView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>720</width>
     <height>450</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="upList">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>50</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>^
^
^
^
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downList">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>260</y>
     <width>50</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>v
v
v
v</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cancelBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
